### ISSUE TITLE
Int/Dis artifacts projection

### ISSUE THREAD

**Issue by SunburnedGoose** [DestinyItemManager/DIM#556]
https://www.reddit.com/r/DestinyTheGame/comments/4kyfnk/let_us_combine_coo_runes_into_the_next_tier_of/d3j4g36?context=3

Because the stat level for these artifacts is 51 instead of 38, is the slope used to forecast the item different?  Users report that their 300 defense artifact was 97% and became 105% at 335 which is higher than our normal margin of error.

**Comment by kyleshay** [comment]
```
        case 'artifact':
          rate = 1/10;
          split = 38;

          scaled = Math.floor(rate * (maxLight - light.value) + stat.base);
```

**Comment by kyleshay** [comment]
fixed (became non-issue) in #572
